MIDIchlorians/MIDIchlorians/SessionName.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  SessionName.swift
+//  MIDIchlorians
+//
+//  Created by Advay Pal on 25/03/17.
+//  Copyright © 2017 nus.cs3217.a0118897. All rights reserved.
+//
+
+import RealmSwift
+
+class SessionName: Object {
+
+    private var sessionName: String = ""
```

Comment:
if the intention is to make a public getter and a private setter we should use `private(set) var sessionString: String = ""`, that way we won't need the getter below